src/Error/Error.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { NavLink } from 'react-router-dom';
+
+const Error = (props) => {
+  return (
+    <section className='error'>
+      <p className='error'>{props ? props.error : 'something went wrong'}</p>
+      <NavLink className='home-nav'>home</NavLink>
+    </section>
```

Comment:
Going to flesh this component out more soon. I am actually not 100% sure I really need it, I will think about it.